doto-frontend/src/components/pages/Settings/SettingsPage.css

```diff
@@ -0,0 +1,63 @@
+.SettingsPage {
+    display: flex;
+    flex-direction: row;
+}
+
+.left-side-bar {
+    border-radius: 0px 79px 0px 0px;
+    width: 20vh;
+    background-color: #3700b3;
+    min-height: 80vh;
+    margin-top: 20vh;
+}
+
+.right-side-bar {
+    border-radius: 79px 0px 0px 0px;
+    width: 95vw;
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    background-color: #8d6cd9;
+    min-height: 80vh;
+    margin-top: 20vh;
+    margin-left: 5vw;
+}
```

Comment:
are these consistent with the login page?

Reply:
I'd sent my .css file to @FocalChord to use the same properties. So the login page would be consistent with this.